modules/mcuboot/CMakeLists.txt

```diff
@@ -167,6 +167,13 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
     SOURCE_DIR ${ZEPHYR_MCUBOOT_MODULE_DIR}/boot/zephyr
     )
 
+  get_shared(multi_image_update IMAGE mcuboot PROPERTY NRF53_MULTI_IMAGE_UPDATE)
```

Comment:
If we add the flash_nop_device's source file in that way, should we also introduce adding a related dts here? Or we want to leave it in applications?

Reply:
Adding it here would let us to keep everything in one place.

Reply:
it's too late to apply a devicetree overlay at this point for the application itself.

That's why we should consider turning the flow around, so that the app has the overlay and a Kconfig setting enabling all of this, and then the app can pass that down to the child image (mcuboot)

Reply:
Thanks for clarification, I will cherry-pick your change in the current state then